.github/workflows/release.yml

```diff
@@ -39,6 +58,7 @@ jobs:
 
   cache-cabal:
     name: 'Cache Cabal'
+    needs: draft-release
     strategy:
```

Comment:
This job must be run on `master` (more generally, on the branch marked as `default` in github) so that all PRs can share the cache.
We have to move it to the `master.yml` file.